### DISCUSSION TITLE
Siemens M60

### DISCUSSION THREAD

**Discussion by ychehntb** [udotdevelopment/ATSPM#113] [Q&A] [answered]
Hi All,

Does anybody have experience integrating Siemens M60 into ATSPM? Could you please share the experiences if any. Specifically, I would like to know following information:

1 Does M60 require FTP or SFTP or SCP?
2 I see in MOE database, there is a default controller type called Siemens SEPAC with username and password defined. Is this the correct type / configuration information for Siemens M60? 
3 In UDOT open source solution, there is a decoder called DecodeSiemensLogs, does this solution work with M60 controllers?

Thank you in advance.

Yi

**Comment by shanejohnson-atspm** [comment]
You should probably contact your local Siemens rep..  They have made significant improvements on the security and reliability of the hi-res  storage and retrieval process.  They would be in the best position to advise you on how to integrate their controllers.

From: Yi Che ***@***.***>
Sent: Wednesday, October 27, 2021 1:07 PM
To: udotdevelopment/ATSPM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [udotdevelopment/ATSPM] Siemens M60 (Issue #101)


Hi All,

Does anybody have experience integrating Siemens M60 into ATSPM? Could you please share the experiences if any. Specifically, I would like to know following information:

1 Does M60 require FTP or SFTP or SCP?
2 I see in MOE database, there is a default controller type called Siemens SEPAC with username and password defined. Is this the correct type / configuration information for Siemens M60?
3 In UDOT open source solution, there is a decoder called DecodeSiemensLogs, does this solution work with M60 controllers?

Thank you in advance.

Yi

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fudotdevelopment%2FATSPM%2Fissues%2F101&data=04%7C01%7CShane.Johnson%40q-free.com%7C0f3357a85d9d4a0bb3c208d9997d01df%7C39ef44caadfa4a248858ef68dbe16345%7C1%7C0%7C637709584458193744%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=q2NS73l5BT%2BC0afZYDVwi6%2B5fgF1o%2B395YyP08KIlXs%3D&reserved=0>, or unsubscribe<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAHPHYUTPP7TU6D6FRSOYZOTUJBEWRANCNFSM5G3CVXDQ&data=04%7C01%7CShane.Johnson%40q-free.com%7C0f3357a85d9d4a0bb3c208d9997d01df%7C39ef44caadfa4a248858ef68dbe16345%7C1%7C0%7C637709584458193744%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ygmJs14qV943qQWUKaSArx8dhgK%2FQVqd9yXi3XrrROE%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7CShane.Johnson%40q-free.com%7C0f3357a85d9d4a0bb3c208d9997d01df%7C39ef44caadfa4a248858ef68dbe16345%7C1%7C0%7C637709584458203732%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=yA5Z7SL%2FVXXDy3Rv68N%2FfpDV2nAiZJr4U%2FcWWaE2Ufo%3D&reserved=0> or Android<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7CShane.Johnson%40q-free.com%7C0f3357a85d9d4a0bb3c208d9997d01df%7C39ef44caadfa4a248858ef68dbe16345%7C1%7C0%7C637709584458203732%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=lLVfdKMZ3aZ1ggX4uYdznA9P9svObBTn3hi%2FZSzykYE%3D&reserved=0>.

**Comment by gault00** [comment, marked as answer]
The m60 uses SFTP and there’s a private key you need to get from Siemens for it to work.
